setup.py

```diff
@@ -358,17 +358,18 @@ def parse_requirements(file_name):
     Copied from cburgmer/pdfserver.
     """
     requirements = []
-    for line in open(file_name, 'r').read().split('\n'):
-        # Ignore comments, blank lines and included requirements files
-        if re.match(r'(\s*#)|(\s*$)|(-r .*$)', line):
-            continue
-
-        if re.match(r'\s*-e\s+', line):
-            requirements.append(re.sub(r'\s*-e\s+.*#egg=(.*)$', r'\1', line))
-        elif re.match(r'\s*-f\s+', line):
-            pass
-        else:
-            requirements.append(line)
+    with open(file_name, 'r') as fh:
+        for line in fh:
+            # Ignore comments, blank lines and included requirements files
+            if re.match(r'(\s*#)|(\s*$)|(-r .*$)', line):
+                continue
+
+            if re.match(r'\s*-e\s+', line):
+                requirements.append(re.sub(r'\s*-e\s+.*#egg=(.*)$', r'\1', line))
+            elif re.match(r'\s*-f\s+', line):
+                pass
+            else:
+                requirements.append(line.rstrip('\n'))
```

Comment:
I prefer this to be changed. That `pass` branch does not help understanding the code in any way.